packages/sdk/src/types/VariableTypes.ts

```diff
@@ -45,6 +45,7 @@ export interface Variable {
     occurrences: number;
     placeholder?: string | null;
     helpText?: string | null;
+    privateData: Record<string, string>;
```

Comment:
Maybe we should have a private data alias?

Reply:
Could be handy